strongbox-storage/strongbox-storage-layout-providers/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Please, remove this `.gitignore` file and use the one in the root directory of the project instead.

Reply:
Should be resolved.